test/parallel/test-runner-coverage-default-exclusion.mjs

```diff
@@ -84,7 +86,7 @@ describe('test runner coverage default exclusion', skipIfNoInspector, () => {
     assert.strictEqual(result.status, 0);
   });
 
-  it('should exclude ts test files when using --experimental-strip-types', async () => {
+  it('should exclude ts test files', async () => {
```

Comment:
noting that this suite never positively affirms typescript coverage - perhaps Node.js does not do that yet and I am just mistaken. I could not tell from the source code or API docs

Reply:
Since --experimental-strip-types is unflag it will also execute the .test.ts files that are present in the folder. So it will change the output

Reply:
that's what confused me - the report on line 90 does not change

Reply:
If you look below --experimental-strip-typed was already enabled